alpha/scripts/utils/updatePartnerEntries2Sphinx.php

```diff
@@ -51,8 +51,16 @@
 		if ($mode == 'execute')
 		{
 			echo "Updating [". $item->getId() ."] ... "; 
-			$sphinx->saveToSphinx($item, false, true);
-			echo "Done\n";
+			try {
```

Comment:
Coding conventions: we set curly brackets at new line
(I would also use the opportunity to align other if statements that don't have (single if line) with curly brackets
at 2011 when script was written it was before we set conventions